subsys/net/lib/lwm2m/lwm2m_rw_json.c

```diff
@@ -910,7 +910,7 @@ int do_write_op_json(struct lwm2m_message *msg)
 	struct lwm2m_engine_res_inst *res_inst = NULL;
```

Comment:
The JSON commit has duplicated `net: lwm2m:` prefix in the title.

Reply:
I will fix that commit message.